### DISCUSSION TITLE
Image 组件中的 img 5px 是没有处理吗？

### DISCUSSION THREAD

**Discussion by moecasts** [ant-design/ant-design#28434] [Q&A] [answered]
这是我的代码，使用了 Layout 中的 Header 组件和 Image 组件。
![image](https://user-images.githubusercontent.com/37169906/102695598-5b8b8d80-4263-11eb-999f-e512ef09c735.png)

问题：`.logo` 的 `line-height` 属性设置为与 `height` 属性同高时，其子元素 `img` 会有一个错位。
![captured](https://user-images.githubusercontent.com/37169906/102695817-b2de2d80-4264-11eb-9332-973d0668a0b8.gif)

解决方法：经过查询是 img 存在的 `5px` 问题，可以通过设置 `.ant-image` 的 `line-height: 1` 解决。

请问是否有在项目代码修改的打算？

**Comment by xrkffgg** [comment, marked as answer]
img 设置 display: block 试试

**Reply by moecasts** [reply]
> img 设置 display: block 试试

设置后也可以解决，请问能更新到组件库的代码中吗？

**Reply by xrkffgg** [reply]
可以来个 PR

**Reply by moecasts** [reply]
好的 :)

**Reply by xrkffgg** [reply]
Fixed in https://github.com/ant-design/ant-design/pull/28439
